openedx/core/djangoapps/theming/helpers_dirs.py

```diff
@@ -107,7 +107,7 @@ def get_project_root_name_from_settings(project_root):
 
 
 @python_2_unicode_compatible
```

Comment:
`python_2_unicode_compatible` should be removed along with its import

Reply:
Removed the `python_2_unicode_compatible` import and statement.